dev/tools/dartdoc.dart

```diff
@@ -90,6 +94,20 @@ dependencies:
   createIndexAndCleanup();
 }
 
+void createFooter(String footerPath) {
+  ProcessResult gitResult = Process.runSync('git', <String>['rev-parse', 'HEAD']);
+  if (gitResult.exitCode != 0)
+    exit(gitResult.exitCode);
+  String gitHead = gitResult.stdout.trim();
+
+  String timestamp = new DateFormat('yyyy-MM-dd HH:mm').format(new DateTime.now());
+
+  new File(footerPath).writeAsStringSync(
+    '<p class="text-center" style="font-size: 10px">'
```

Comment:
The dartdoc generated page already contains a <footer>.  This text will be placed above that.